cumulusci/core/config/project_config.py

```diff
@@ -266,15 +266,15 @@ def repo_root(self):
         if path:
             return path
 
-        path = Path(os.path.splitdrive(Path.cwd())[1])
+        path = Path.cwd()
         while True:
             if (path / ".git").is_dir():
                 return str(path)
-            head, tail = os.path.split(path)
-            if not tail:
+            parent = path.parent
+            if parent == path:
                 # reached the root
                 break
-            path = Path(head)
+            path = parent
```

Comment:
The docs for PurePath.parent say: "If you want to walk an arbitrary filesystem path upwards, it is recommended to first call Path.resolve() so as to resolve symlinks and eliminate “..” components."

Reply:
6fad035